### DISCUSSION TITLE
How can i customize the theme of and extention loaded by telescope

### DISCUSSION THREAD

**Discussion by theSafdarAwan** [nvim-telescope/telescope.nvim#1416] [Q&A]
currently i was reading the doc's and found this i don't know what tj was trying to say but
 ```txt
 telescope.extensions()                                *telescope.extensions()*
    Use telescope.extensions to reference any extensions within your
    configuration.
    While the docs currently generate this as a function, it's actually a
    table. Sorry.
```
how can i change the behaviour of the extention

**Comment by l-kershaw** [comment]
I don't really understand what your aim is here.

Are you trying to get a picker provided by an extension to use a given theme (as in `:h telescope.themes`)?
If so you can pass the chosen `theme` when calling the extension, e.g.:
```viml
:Telescope project project theme=get_ivy
```
```lua
require("telescope").extensions.project.project(require("telescope.themes").get_ivy())
```

Are you trying to configure how a particular extension works?
If so, this will depend on the extension, and you should make an issue/discussion in the repo for the extension.
